<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RestorantTakip</class>
 <widget class="QWidget" name="RestorantTakip">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>280</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RestorantTakip</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>80</y>
     <width>271</width>
     <height>161</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_9">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_23">
      <item>
       <widget class="QPushButton" name="pMod">
        <property name="text">
         <string>%</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pWipe">
        <property name="text">
         <string>Temizle</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pPN">
        <property name="text">
         <string>+/-</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_24">
      <item>
       <widget class="QPushButton" name="p1divX">
        <property name="text">
         <string>1/X</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pxSq">
        <property name="text">
         <string>X^2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pRootSq">
        <property name="text">
         <string>2|x</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pdiv">
        <property name="text">
         <string>/</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_27">
      <item>
       <widget class="QPushButton" name="p7">
        <property name="text">
         <string>7</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="p8">
        <property name="text">
         <string>8</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="p9">
        <property name="text">
         <string>9</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pX">
        <property name="text">
         <string>X</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_26">
      <item>
       <widget class="QPushButton" name="p4">
        <property name="text">
         <string>4</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="p5">
        <property name="text">
         <string>5</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="p6">
        <property name="text">
         <string>6</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pMin">
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_25">
      <item>
       <widget class="QPushButton" name="p1">
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="p2">
        <property name="text">
         <string>2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="p3">
        <property name="text">
         <string>3</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pPos">
        <property name="text">
         <string>+</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_28">
      <item>
       <widget class="QPushButton" name="pdot">
        <property name="text">
         <string>.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="p0">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pDel">
        <property name="text">
         <string>DEL</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pequals">
        <property name="text">
         <string>=</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>241</width>
     <height>236</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_sip">
        <property name="text">
         <string>Sipariş No</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_kiz">
        <property name="text">
         <string>Kızartma</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_ogl">
        <property name="text">
         <string>Öğle Yemeği</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_bur">
        <property name="text">
         <string>Burger</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_piz">
        <property name="text">
         <string>Pizza</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_che">
        <property name="text">
         <string>Cheese Burger</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_ice">
        <property name="text">
         <string>İçecekler</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLineEdit" name="lin_sip"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lin_kiz">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lin_ogl">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lin_bur">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lin_piz">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lin_che">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lin_ice">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>10</y>
     <width>261</width>
     <height>231</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_fiy">
        <property name="text">
         <string>Fiyat</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_ser">
        <property name="text">
         <string>Servis Hizmeti</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_kdv">
        <property name="text">
         <string>%18 KDV</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_alt">
        <property name="text">
         <string>Alt Toplam</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_top">
        <property name="text">
         <string>Toplam</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLineEdit" name="lin_fiy"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lin_ser"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lin_kdv"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lin_alt"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lin_top"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>220</y>
     <width>340</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QPushButton" name="push_fiyat">
      <property name="text">
       <string>Fiyatlar</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="push_toplam">
      <property name="text">
       <string>Toplam</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="push_sifirla">
      <property name="text">
       <string>Sıfırla</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="push_cikis">
      <property name="text">
       <string>Çıkış</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>40</y>
     <width>61</width>
     <height>202</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QPushButton" name="eksi">
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="eksi_2">
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="eksi_3">
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="eksi_4">
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="eksi_5">
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="eksi_6">
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QPushButton" name="arti">
        <property name="text">
         <string>+</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="arti_2">
        <property name="text">
         <string>+</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="arti_3">
        <property name="text">
         <string>+</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="arti_4">
        <property name="text">
         <string>+</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="arti_5">
        <property name="text">
         <string>+</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="arti_6">
        <property name="text">
         <string>+</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="sonuc">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>11</y>
     <width>271</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <bold>true</bold>
     <underline>true</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
